packages/zosfiles/__tests__/__system__/methods/download/__resources__/download_file.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+account=$1
+host=$2
+port=$3
+user=$4
+password=$5
+ru=$6
+fileToDownload=$7
+attributes=$8
+
+zowe zos-files download uss $fileToDownload --attributes $attributes --host $host --port $port --user $user --password $password --ru $ru
```

Comment:
Please put quotes around the `$attributes` and `$fileToDownload`, otherwise system tests will fail if they run in an environment with spaces in the path.